style.css

```diff
@@ -51,7 +51,7 @@
     background-color: #e5e5e5;
     background-image: url(./bg/5.png), url(./bg/9.png), url(./bg/3.png), url(./bg/1.png), url(./bg/7.png), url(./bg/4.png), url(./bg/8.png), url(./cards/Frame.png), url(./cards/Shape.png);
     background-repeat: no-repeat;
-    background-position: 100% 40rem, 100% 17rem, 100% 3rem, 50% 3rem, 0% 32rem, 30% 33.5rem, 5% 11rem, 100% 300rem, 50% 99.8%;
+    background-position: 100% 40rem, 100% 17rem, 100% 3rem, 50% 3rem, 0% 32rem, 30% 33.5rem, 5% 11rem, 100% 97%, 50% 99.8%;
   }
```

Comment:
Please note the height of the `Get my Resume` button height in design and fix yours accordingly.
![resume-btn](https://user-images.githubusercontent.com/35504149/129954752-9c6fa65d-418e-4f09-b22a-835a6c83ddc7.png)